pkgs/development/libraries/pstreams/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv
+, fetchgit
+}:
+
+stdenv.mkDerivation rec {
+  pname = "PStreams";
+  version = "d21afef311be496e6495a5e2103eec7a53ecb817";
+
+  src = fetchgit {
+    url = https://git.code.sf.net/p/pstreams/code;
+    rev = version;
+    sha256 = "1pqb2mbd36fq3ay1ngmpcqz9ajcj8hcqlkk4a7r02sxv94415hgh";
+  };
+
+  makeFlags = [ "prefix=$(out)" ];
+  dontBuild = true;
+  doCheck = true;
+
+  meta = with stdenv.lib; {
+    description = "POSIX Process Control in C++";
+    longDescription = ''
+      PStreams allows you to run another program from your C++ application and
+      to transfer data between the two programs similar to shell pipelines.
+
+      In the simplest case, a PStreams class is like a C++ wrapper for the
```

Comment:
This description is way too long.
long descriptions usually look like this https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/compilers/gcc/9/default.nix#L364

Reply:
We don't really set a maximum length [in the manual](https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes). We have other packages with truly long `longDescription` (cf factorio or hedgewars) and it still renders properly on nixos.org package listing.

But here we can probably lose the last paragraph and still retain the essential bits of information.

Reply:
Agreed. No point providing internal details here.